model/project_ref.go

```diff
@@ -745,22 +747,15 @@ func (p *ProjectRef) AttachToNewRepo(ctx context.Context, u *user.DBUser) error
 		ProjectRefRepoRefIdKey: p.RepoRefId,
 	}
 	update = p.addGithubConflictsToUpdate(update)
+
 	err = db.UpdateId(ProjectRefCollection, p.Id, bson.M{
-		"$set": update,
+		"$set":   update,
+		"$unset": bson.M{ProjectRefTracksPushEventsKey: 1},
 	})
 	if err != nil {
 		return errors.Wrap(err, "updating owner/repo in the DB")
 	}
-	_, err = EnableWebhooks(ctx, p)
```

Comment:
i might be confused but was there a need to get rid of enabling webhooks here?

Reply:
I switched to unsetting it because we can rely on the repo's value.